kiwi/schema/kiwi.rnc

```diff
@@ -2794,7 +2794,7 @@ div {
         ## expired. In countdown mode an indication of the remaining time
         ## is displayed. The attribute is available for the grub loader.
         attribute timeout_style {
-            "countdown" | "hidden"
+            "countdown" | "hidden" | "menu"
```

Comment:
One of the non-whitespace changes is here (for reviewers)

Reply:
I separated the original commit into two commits. One for the actual (non-whitespace) change and another commit for only the whitespace changes (which admittedly isn't that important)
This commit doesn't contain whitespace changes.
https://github.com/OSInside/kiwi/pull/2341/commits/7627f2c677052dfe7ab00b3a213dc99c125a120b